SDL/LST.h

```diff
@@ -9,6 +9,11 @@
 #include "LSTESData.h"
 #endif
 
+#include "Event.h"
+
+#include "Math/Vector3D.h"
```

Comment:
can these be avoided?
Especially the first one; to keep the dependencies cleaner.

Reply:
These were not added, they were just moved from the `.cc` to the `.h` file. Unless the movement is the problem.

Reply:
> Unless the movement is the problem.

yes

Reply:
Sure, I will revert.

Reply:
Reverted in [37b465a](https://github.com/SegmentLinking/TrackLooper/pull/408/commits/37b465af7330a92688c3ceb58416e25d7946b871). I also included back the explicit `SDL/Event.h` dependency for `sdl.h` to make it clearer, even though it is implicitly included through `write_sdl_ntuple.h`.